.circleci/config.yml

```
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  build_docs:
```

```suggestion
  docs:
```
So that we don't have to change the name of the expected check.